api/proto/lightnode/lightnode.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package lightnode;
+import "common/common.proto";
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/lightnode";
+
+///////////////////////////////////////////////////////////////////////////////////
+// Warning: light node APIs are currently in flux. This is not yet a stable API. //
+///////////////////////////////////////////////////////////////////////////////////
+
+service LightNode {
+	// GetChunk retrieves a specific chunk held by the light node.
+	rpc GetChunk(GetChunkRequest) returns (GetChunkReply) {}
+
+	// StreamAvailabilityStatus streams the availability status of all chunks assigned to the light node.
+	// For use by a DA node for monitoring the availability of chunks through its constellation of agent light nodes.
+	rpc StreamAvailabilityStatus(StreamAvailabilityStatusRequest) returns (stream StreamAvailabilityStatusReply) {}
+}
+
+// A request for a specific chunk from a light node.
+message GetChunkRequest {
+	bytes header_hash = 1;
+}
+
+// A reply to a GetChunk request.
+message GetChunkReply {
+	// The chunk data.
+	common.ChunkData chunk = 1;
+}
+
+// A request from a DA node to an agent light node to stream the availability status of all chunks
+// assigned to the light node.
+message StreamAvailabilityStatusRequest {
+	bytes authentication_token = 1;
+}
+
+// A reply to a StreamAvailabilityStatus request.
+message StreamAvailabilityStatusReply {
+	bytes header_hash = 1;
```

Comment:
The comment of the API says "availability status of all chunks", but this is a blob header hash, how should users interpret it? (document it here)

Reply:
Updated doc as follows:

```
	// The hash of a blob header corresponding to a chunk the agent received and verified. From the light node's
	// perspective, the blob is available if the chunk is available.
	bytes header_hash = 1;
```

Reply:
This comment is helpful. Note my point was actually related to the naming of the API method itself, i.e. the return value is "blobs (header hash) that are available at the light node", not "availability statuses of all blobs that the light node received".

Reply:
Updated RPC name and comment as follows:

```
	// StreamBlobAvailability streams the availability status blobs from the light node's perspective.
	// A light node considers a blob to be available if all chunks it wants to sample are available.
	// This API is for use by a DA node for monitoring the availability of chunks through its 
	// constellation of agent light nodes.
	rpc StreamBlobAvailability(StreamChunkAvailabilityRequest) returns (stream StreamChunkAvailabilityReply) {}
```